source/CCustomOpcodeSystem.cpp

```diff
@@ -3098,6 +2665,23 @@ extern "C"
 		return result;
 	}
 
+	void WINAPI CLEO_ReadStringParamWriteBuffer(CLEO::CRunningScript* thread, char** outBuf, int* outBufSize, DWORD* outNeedsTerminator)
+	{
+		if (thread == nullptr || 
+			outBuf == nullptr || 
+			outBufSize == nullptr ||
+			outNeedsTerminator == nullptr)
+		{
+			LOG_WARNING(thread, "Invalid argument of CLEO_ReadStringParamWriteBuffer in script %s\nScript suspended.", ((CCustomScript*)thread)->GetInfoStr().c_str());
+			return;

```

Comment:
warning says "script suspended" but it does not happen?

Reply:
True! Fixed